rules/enforce-store-naming-convention/enforce-store-naming-convention.js

```diff
@@ -38,11 +45,24 @@ module.exports = {
 
           const storeName = node.id.name;
 
-          if (storeName?.startsWith("$")) {
+          if (mode === "prefix" && storeName?.startsWith("$")) {
```

Comment:
Shall we extract this logic to some place? I see, it is used here and in the JS-case. Also, I suppose it should be used in `effector/no-getState` JS-case (it detects stores by naming convention).

Reply:
I think in this case we have to change the implementation and use [shared settings](https://eslint.org/docs/user-guide/configuring/configuration-files#adding-shared-settings). Otherwise, it wouldn’t be possible to share the mode value across several rules.

What do you think?

Reply:
It seems like the good idea 👍